### ISSUE TITLE
include Cargo.lock

### ISSUE THREAD

**Issue by docteurklein** [mkaski/pg_render#1]
Hello and thanks for this extension, very useful!

I wanted to package this in nix, but the absence of `Cargo.lock` in git complicated the process (https://github.com/docteurklein/httpg/blob/8b9f2de81292dff2adfd4de9af32ac60565d87ce/flake.nix#L37-L39).

Is there a specific reason for not including it?  
I read:
 - https://doc.rust-lang.org/cargo/faq.html#why-have-cargolock-in-version-control
 - https://doc.rust-lang.org/cargo/commands/cargo-install.html#manifest-options
 - https://doc.rust-lang.org/cargo/commands/cargo-install.html#dealing-with-the-lockfile

which states that the lock file will be ignored by cargo unless explicitly given a `--locked` flag to `cargo build` for example,
which is what you do already in the GH workflows and in the README, so it means you have an unversionned lock file anyway, AFAICT.

In conclusion, adding the lock file to git would simplify everyone's life apparently :)

Just wanted to check if it's OK with you before sending an eventual PR or being convinced otherwise.

In any case, have a good day :)

**Comment by docteurklein** [comment]
here is an example of what this would look like: https://github.com/mkaski/pg_render/pull/2

**Comment by docteurklein** [comment]
I agree, a separate change for an eventual nix flake is surely better

**Comment by mkaski** [comment]
Cargo.lock added and deps updated
https://github.com/mkaski/pg_render/releases/tag/v0.1.2
